mypy/stubgen.py

```diff
@@ -91,15 +92,49 @@
 from mypy.traverser import has_return_statement
 
 
+# Common ways of naming package containing vendored modules.
+VENDOR_PACKAGES = [
+    'packages',
+    'vendor',
+    'vendored',
+]
+
+# Avoid some file names that are unnecessary or likely to cause trouble (\n for end of path).
+BLACKLIST = [
+    '/six.py\n',  # Likely vendored six; too dynamic for us to handle
```

Comment:
Maybe make these regexps and use `$` instead of `\n`?

Reply:
I'll deal with this in a follow-up PR, since the change would cause a merge conflict.